nemo/backends/torch_backend.py

```diff
@@ -0,0 +1,64 @@
+# =============================================================================
+# Copyright (c) 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+from os.path import expanduser
+from typing import Any, Dict
+
+import torch
+
+
+def save(checkpoint: Dict[str, Any], filename: str) -> None:
+    """
+        A proxy function that saves the checkpoint to a given file.
```

Comment:
Indentation should be reduced by 1 tab size.

Reply:
ech, but I like this indent in comments... ;)

Reply:
done